wsgi/odyssy/basic/views.py

```diff
@@ -1,5 +1,9 @@
+import os
+
 from django.shortcuts import render
 
+from events.models import Event
 
 def index(request):
-    return render(request, 'basic/index.html')
+    events_list = Event.objects.order_by('-event_date')[:int(os.environ.get('NUM_EVENTS_ON_INDEX_PAGE', 3 ))]
```

Comment:
By configurable, I meant that there should be a function that is able to call it and get results. Please create a `staticmethod` (maybe `get_latest_events`) that would return latest `n` events based on start date. This would make things easy to test and also avoid errors that would be introduced by writing it again and again.

Make such functions for everything so that we do not need to write them again while making other services.